Dockerfile

```diff
@@ -0,0 +1,39 @@
+# based on https://stackoverflow.com/questions/53835198/integrating-python-poetry-with-docker
+FROM python:3.8-alpine3.11 as base
```

Comment:
This should be `python:3.8-slim`. Using alpine forces musl which requires some wheels to be compilled. Saving 50MB, for worst performance is not worth it.